dynamic_bitset.html

```diff
@@ -579,10 +598,7 @@ <h3><a id="constructors">Constructors</a></h3>
  <b>Postconditions:</b> <tt>this-&gt;size() == 0</tt>.<br />
  <b>Throws:</b> An allocation error if memory is exhausted
```

Comment:
This "Throws:" can be removed or be changed to "Throws: Nothing". This is because this constructor should not throw any exceptions. No Allocator is being constructed here, only copied, and copying Allocators is required to not throw exceptions.